core/trino-main/src/main/java/io/trino/sql/relational/SqlToRowExpressionTranslator.java

```diff
@@ -346,6 +348,25 @@ protected RowExpression visitFunctionCall(FunctionCall node, Void context)
             return new CallExpression(metadata.decodeFunction(node.getName()), arguments);
         }
 
+        @Override
+        protected RowExpression visitTrim(Trim node, Void context)
```

Comment:
This rewriter is only used in the Planner. Do not support `Trim` here.

Reply:
`assertFunction` used in `TestStringFunctions` depends on `SqlToRowExpressionTranslator` if I understand correctly. Let me remove these code and fix tests instead.